### ISSUE TITLE
install.ps1 is not signed

### ISSUE THREAD

**Issue by salemoh** [Microsoft/CNTK#1278]
Setting RemoteSigned still doesn't allow the powershell to run the install script. The only way to run it is through Unrestricted.

Is this by design?

**Comment by mahilleb-msft** [comment]
We advise to unblock the .zip before extracting and running the script from your local computer. RemoteSigned should be enough for that case (and no, we currently don't sign the script).
https://github.com/Microsoft/CNTK/wiki/Setup-Windows-Binary-Script

Does this clarify? If not, could you provide repro steps?

Thanks, Mark

**Comment by salemoh** [comment]
Thanks Mark. It wasn't clear that that blocking described in the instructions was about the execution policy in Powershell. Would be great to add to the FAQ that if you get an error about unsigned script this is due to the ZIP file step.

For me I installed CNTK using unrestricted policy in elevated Powershell

**Comment by wolfma61** [comment]
this has been fixed a while ago ...
please open a new issue if you experience this in a more recent CNTK installation
